### DISCUSSION TITLE
单机多卡微调命令是否有误

### DISCUSSION THREAD

**Discussion by 3553531421** [zai-org/ChatGLM3#1208] [Q&A] [answered]
![image](https://github.com/THUDM/ChatGLM3/assets/139878246/62c82d2e-07ad-474e-bf90-4cd15a5119e5)
因为加了configs/ds_zero_2.json，所以这个参数就被读成了auto_resume_from_checkpoint
![image](https://github.com/THUDM/ChatGLM3/assets/139878246/76abf0f7-cd5e-4ad4-8d33-1254c10d04d7)
所以就结束了，没有报错，不加configs/ds_zero_2.json又要内存不够

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
看一下教程 加入这个参数前面还需要加上 deepspeed
